jwt-bearer-auth-yesod/src/Web/Auth/Bearer/JWT/Yesod/Lens.hs

```diff
@@ -1,23 +1,19 @@
 -- |
 -- This module provides JWT-specific lenses and typeclasses for Yesod applications.
 module Web.Auth.Bearer.JWT.Yesod.Lens
-  ( HasJWKStore (..)
-  , handlerJWKStoreL
+  ( handlerCacheWithSettingsL
   ) where
 
 import Prelude
 
 import Control.Lens
+import Web.Auth.Bearer.JWT.Yesod.Types
 import Yesod.Core.Types
 import Yesod.Core.Types.Lens
 
--- | Typeclass for applications that have a JWK store
--- The store type is polymorphic to allow different JWK store implementations
-class HasJWKStore store app where
-  jwkStoreL :: Lens' app store
-
--- | Composed lens from HandlerData to JWK store
--- Goes: HandlerData -> RunHandlerEnv -> site -> JWK store
-handlerJWKStoreL
-  :: HasJWKStore store site => Lens' (HandlerData child site) store
-handlerJWKStoreL = handlerEnvL . rheSiteL . jwkStoreL
+-- | Composed lens from HandlerData to JWT Bearer auth settings
+-- Goes: HandlerData -> RunHandlerEnv -> site -> JWT Bearer auth settings
+handlerCacheWithSettingsL
+  :: HasJWKCacheSettings site
+  => Lens' (HandlerData child site) CacheWithSettings
+handlerCacheWithSettingsL = handlerEnvL . rheSiteL . jwkCacheSettingsL
```

Comment:
This doesn't seem particularly useful vs just in-lining it wherever used. Then you can ditch the whole module.

Reply:
done